### PR TITLE
feat(explorer): Home page Displays Recent Batches with Information

### PR DESCRIPTION
**Pull request by PatStiles**
# Home Page Displays Recent Batches with Information

## Description

The past home page displayed only the batch merkle root of a batch. This pr adds additional additional batch information: `Batch Hash`, `Status`, `Age`, and `Submission Block Number`. It also adds an optional header field for the table component to be used in a larger refactor.

Screenshot 2024-12-18 at 04 45 47

## To Test:
- Start a local devnet
- Run the explorer and observe the recent batches displayed on the homepage match the ones on the batcher page.

## Type of change

Please delete options that are not relevant.

- [x] New feature

## Checklist

- [ ] “Hotfix” to `testnet`, everything else to `staging`
- [ ] Linked to Github Issue
- [ ] This change depends on code or research by an external entity
  - [ ] Acknowledgements were updated to give credit
- [ ] Unit tests added
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] This change is an Optimization
  - [ ] Benchmarks added/run
- [ ] Has a known issue
  - [Link to the open issue addressing it]() 
- [ ] If your PR changes the Operator compatibility (Ex: Upgrade prover versions)
  - [ ] This PR adds compatibility for operator for both versions and do not change batcher/docs/examples
  - [ ] This PR updates batcher and docs/examples to the newer version. This requires the operator are already updated to be compatible

### CODE REVIEW

File: explorer/lib/explorer_web/live/pages/home/index.html.heex [hunk 1]
```diff
@@ -40,30 +40,44 @@
...
     </.card>
     <.live_component module={ContractsComponent} id="contracts_card" class="sm:col-span-2" />
-    <%= if @latest_batches != :empty and @latest_batches != [] do %>
-      <.card
-        class="relative sm:col-span-2 w-full flex flex-col space-y-1"
-        inner_class="text-ellipsis overflow-hidden text-lg flex flex-col space-y-1"
-        title="Recent Batches"
-      >
+    <div class="relative truncate sm:col-span-2">
+        <h1 class="text-4xl sm:text-4xl font-bold font-foreground text-left py-2">
```
Comment (MarcosNicolau) [inline explorer/lib/explorer_web/live/pages/home/index.html.heex]: Nice, I would only make the header smaller, try:
```suggestion
        <h1 class="text-xl font-bold font-foreground text-left py-2">
```

**Review by MarcosNicolau — APPROVED** [review]

File: explorer/lib/explorer_web/live/pages/batches/index.html.heex
Comment (uri-99) [inline explorer/lib/explorer_web/live/pages/batches/index.html.heex:1]: image

File: explorer/lib/explorer_web/live/pages/home/index.html.heex (hunk 1 above)
Comment (PatStiles) [reply explorer/lib/explorer_web/live/pages/home/index.html.heex]: addressed!

File: explorer/lib/explorer_web/live/pages/batches/index.html.heex
Comment (PatStiles) [reply explorer/lib/explorer_web/live/pages/batches/index.html.heex:1]: good catch!

**Review by MarcosNicolau — APPROVED** [review]

**Review by uri-99 — APPROVED** [review]

**Pull request merged**